### PR TITLE
Translate your-first-component.md

### LINKED ISSUE #81: Your First Component

### PR DESCRIPTION
**Pull request by rammba**
Closes #81

### CODE REVIEW

File: src/content/learn/describing-the-ui.md [hunk 1]
```diff
@@ -4,19 +4,19 @@ title: Opisivanje korisničkog interfejsa (UI)
 
 <Intro>
 
-React je JavaScript biblioteka za prikazivanje korisničkog interfejsa (UI). UI se sastoji od malih jedinica kao što su dugmad, tekst i slike. React vam omogućava da ih kombinujete u ponovljive, nested (ugnježdene) *komponente.* Od veb sajtova do telefonskih aplikacija, sve na ekranu može se razbiti na komponente. U ovom poglavlju naučićete kako da kreirate, prilagodite i uslovno prikažete React komponente.
+React je JavaScript biblioteka za prikazivanje korisničkog interfejsa (UI). UI se sastoji od malih jedinica kao što su dugmad, tekst i slike. React vam omogućava da ih kombinujete u reusable, nested (ugnježdene) *komponente.* Od web sajtova do telefonskih aplikacija, sve na ekranu može se razbiti na komponente. U ovom poglavlju naučićete kako da kreirate, prilagodite i uslovno prikažete React komponente.
```
Comment (rammba) [inline src/content/learn/describing-the-ui.md:7]: I'm not sure if we should use `reusable` term, but for me, `ponovljivo` doesn't sound as the appropriate translation in this context. Do you have any better idea?

File: src/content/learn/your-first-component.md [hunk 2]
```diff
@@ -152,95 +152,93 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 
-When a child component needs some data from a parent, [pass it by props](/learn/passing-props-to-a-component) instead of nesting definitions.
+Kada detetu (child) komponenti trebaju podaci od roditelja (parent), [prosledite ih preko props-a](/learn/passing-props-to-a-component) umesto da ugnježdavate definicije.
 
 </Pitfall>
 
 <DeepDive>
 
-#### Components all the way down {/*components-all-the-way-down*/}
+#### Komponente nemaju granice {/*components-all-the-way-down*/}
```
Comment (rammba) [inline src/content/learn/your-first-component.md:212]: This doesn't sound as the best translation, but I don't know any better way for `all the way down` phrase.

**Review by lxmarinkovic — APPROVED** [review]
Looks correct, and I have decided not to be picky in this stage as you are doing a great chunk of work and that should reflect on overall progress here. Details could be discussed later.
Thanx for great effort! 🚀

File: src/content/learn/your-first-component.md (hunk 2 above)
Comment (lxmarinkovic) [reply src/content/learn/your-first-component.md:212]: leet it be as it is for now :D

**Pull request merged**